python_modules/libraries/dagster-dbt/tox.ini

```diff
@@ -23,6 +23,6 @@ allowlist_externals =
   /bin/bash
 commands =
   !windows: /bin/bash -c '! pip list --exclude-editable | grep -e dagster'
-  cloud: pytest --reruns 2 --durations 10 -c ../../../pyproject.toml -m "cloud" -vv {posargs}
-  core: pytest --reruns 2 --durations 10 -c ../../../pyproject.toml -m "not legacy and not cloud" -vv {posargs}
-  legacy: pytest --reruns 2 --durations 10 -c ../../../pyproject.toml -m "legacy" -vv {posargs}
+  cloud: pytest --numprocesses 6 --durations 10 -c ../../../pyproject.toml -m "cloud" -vv {posargs}
+  core: pytest --numprocesses 6 --durations 10 -c ../../../pyproject.toml -m "core" -vv {posargs}
```

Comment:
howd you pick 6?

Reply:
`--numprocesses auto` results in `2` for the Buildkite medium queue of machines. Thought we could juice it up a bit more, so just manually increased it to 6. Nothing too scientific.